### PR TITLE
vnet: Add kuskus outbound network dependency

### PR DESCRIPTION
**Pull request by basaba**

### CODE REVIEW

**Comment by cosh** [comment]
#sign-off

**Comment by shsagir** [comment]
#sign-off

**Comment by Court72** [comment]
@shsagir - Can you confirm that the IP address on line 380 is fictitious? If it isn't, can you update it to follow the [Microsoft Learn legal guidelines for IP addresses](https://review.learn.microsoft.com/en-us/help/contribute/contribute-legal-guidelines?branch=main#ip-addresses)?

#hold-off

**Comment by Court72** [comment]
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/vnet-deployment.md [hunk 1]
```diff
@@ -363,7 +363,28 @@ The [ARMClient](https://chocolatey.org/packages/ARMClient) is used to demonstrat
...
+            "location": "Australia Central",
+            "properties": {
+              "category": "Internal Tracing",
+              "endpoints": [
+                {
+                  "domainName": "ingest-kuskuseaau.australiaeast.kusto.windows.net",
+                  "endpointDetails": [
+                    {
+                      "port": 443,
+                      "ipAddress": "20.70.218.111"
```
Comment (cosh) [inline data-explorer/vnet-deployment.md:380]: why don't we change it to 25.24.23.22

File: data-explorer/vnet-deployment.md [hunk 2]
```diff
@@ -363,7 +363,28 @@ The [ARMClient](https://chocolatey.org/packages/ARMClient) is used to demonstrat
...
+          {
+            "id": "/subscriptions/<subscriptionId>/resourceGroups/<resourceGroup>/providers/Microsoft.Kusto/Clusters/<clusterName>/OutboundNetworkDependenciesEndpoints/InternalTracing",
+            "name": "<clustername>/InternalTracing",
+            "type": "Microsoft.Kusto/Clusters/OutboundNetworkDependenciesEndpoints",
+            "location": "Australia Central",
+            "properties": {
+              "category": "Internal Tracing",
+              "endpoints": [
+                {
+                  "domainName": "ingest-kuskuseaau.australiaeast.kusto.windows.net",
```
Comment (cosh) [inline data-explorer/vnet-deployment.md:376]: let's change that to "ingest-<internalTracingCluster>.<region>.kusto.windows.net"

File: data-explorer/vnet-deployment.md (hunk 1 above)
Comment (shsagir) [inline data-explorer/vnet-deployment.md:380]:
```suggestion
                      "ipAddress": "25.24.23.22"
```

**Pull request merged**